.buildkite/pipeline.yml

```diff
@@ -44,16 +44,24 @@ steps:
       build.env("K8S_NIGHTLY") == "1"
     steps:
       - key: charts-tests
-        label: Charts tests
+        label: Charts tests {{matrix.testpackagename}}
```

Comment:
Why does this need to be duplicated?

Reply:
I don't understand what is duplicated?

Reply:
oooooh, I see. Github's elision was playing tricks on me. The matrix is applied at the `step` level instead of the group level. It looks like the test step had two copies for some reason.

Reply:
Those are 2 distinct steps. One run test and another is parsing junit output